ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -1195,8 +1195,7 @@ objects:
 
         # begin bootstrapping
         if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
-          # Derived from https://github.com/openshift/installer/blob/93c78d09ed9e2badb4bf5dab708152fe6b3b6602/data/data/rhcos.json#L40
-          RHCOS_AMI=ami-0aea6a5be0fc2b3fc  # FIXME: assumes AWS_REGION is us-east-1
+          RHCOS_AMI="$(jq -r .amis[\"${AWS_REGION}\"].hvm /var/lib/openshift-install/rhcos.json)"
```

Comment:
I'd have expected:

```sh
RHCOS_AMI="$(jq -r ".amis[\"${AWS_REGION}\"].hvm" /var/lib/openshift-install/rhcos.json)"
```

(with additional quoting around the `jq` expression) but this seems to work as you have it as well.